rust/datafusion/benches/sort_limit_query_sql.rs

```diff
@@ -66,21 +66,23 @@ fn create_context() -> ExecutionContext {
     )
     .unwrap();
 
-    let mem_table = MemTable::load(&csv).unwrap();
+    // let mem_table = MemTable::load(&csv).await.unwrap();
+    //
+    // // create local execution context
+    // let ctx = ExecutionContext::new();
+    // ctx.state.config.concurrency = 1;
+    // ctx.register_table("aggregate_test_100", Box::new(mem_table));
+    // Arc::new(Mutex::new(ctx))
 
-    // create local execution context
-    let mut ctx = ExecutionContext::new();
-    ctx.state.config.concurrency = 1;
-    ctx.register_table("aggregate_test_100", Box::new(mem_table));
-    ctx
+    unimplemented!()
```

Comment:
🤔

Reply:
working on this now ... about to push the fix

Reply:
The issue was that I couldn't call async code from criterion so I had to create a separate tokio runtime and block on some async code. I expect that there may be a cleaner way to do this.